File: ch07/VbServerControl1/Listing07-35.vb
Imports System.Web.UI
Imports System.Web.UI.WebControls

Friend Class DefaultMessageTemplate
    Implements ITemplate

    Public Sub InstantiateIn(ByVal container As System.Web.UI.Control) _
            Implements System.Web.UI.ITemplate.InstantiateIn

        Dim l As New Literal()
        l.Text = "No MessageTemplate was included."
        container.Controls.Add(l)
    End Sub
End Class

File: ch07/CSharpServerControl1/Listing07-35.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace CSharpServerControl1
{
    internal sealed class DefaultMessageTemplate : ITemplate
    {
        public void InstantiateIn(Control container)
        {
            Literal l = new Literal();
            l.Text = "No MessageTemplate was included.";
            container.Controls.Add(l);
        }
    }
}